tests/publiccloud/instance_overview.pm

```diff
@@ -63,7 +63,7 @@ sub collect_system_information {
     assert_script_run("cat /proc/cpuinfo | tee instance_overview/cpuinfo.txt");
     assert_script_run("cat /proc/meminfo | tee instance_overview/meminfo.txt");
     assert_script_run("uname -a | tee instance_overview/uname.txt");
-    tar_and_upload_log("instance_overview/", "instance_overview.tar.gz");
+    tar_and_upload_log("instance_overview/", "instance_overview.tar.gz" {gzip => 1});
```

Comment:
have you missed the come or it is some yet-another weird perl syntax `"instance_overview.tar.gz" {gzip => 1}` :thinking: 
```suggestion
    tar_and_upload_log("instance_overview/", "instance_overview.tar.gz", {gzip => 1});
```